cc/dual_net/factory.cc

```diff
@@ -168,4 +72,13 @@ std::unique_ptr<DualNetFactory> NewDualNetFactory(std::string model_path,
   return nullptr;
 }
 
+}  // namespace
+
+DualNetFactory::~DualNetFactory() = default;
+
+std::unique_ptr<DualNetFactory> NewDualNetFactory(
```

Comment:
This adds adds a batching wrapper around every DualNet implementation, even for cases where it doesn't make sense (e.g. some accelerators only support a batch size of 1).